src/Maestro/Maestro.MergePolicies/ForwardFlowMergePolicy.cs

```diff
@@ -0,0 +1,171 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Maestro.MergePolicyEvaluation;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.DarcLib.Models.VirtualMonoRepo;
+
+namespace Maestro.MergePolicies;
+internal class ForwardFlowMergePolicy : MergePolicy
+{
+    public override string DisplayName => "ForwardFlow";
+
+    private static readonly string configurationErrorsHeader = """
+         ### :x: Check Failed
+
+         The following error(s) were encountered:
+
+
+        """;
+
+    private static readonly string seekHelpMessage = """
+
+        ### :exclamation: IMPORTANT
+
+        The source-manifest.json and Version.Details.xml files are managed by the Maestro bot. Outside of exceptional circumstances, these files should not be modified manually.
+        **Unless you are sure that you know what you are doing, we recommend reaching out for help**. You can receive assistance by:
+        - tagging the @dotnet/product-construction team in a PR comment
+        - using the [First Responder channel](https://teams.microsoft.com/l/channel/19%3Aafba3d1545dd45d7b79f34c1821f6055%40thread.skype/First%20Responders?groupId=4d73664c-9f2f-450d-82a5-c2f02756606dhttps://teams.microsoft.com/l/channel/19%3Aafba3d1545dd45d7b79f34c1821f6055%40thread.skype/First%20Responders?groupId=4d73664c-9f2f-450d-82a5-c2f02756606d),
+        - [opening an issue](https://github.com/dotnet/arcade-services/issues/new?template=BLANK_ISSUE) in the dotnet/arcade-services repo
+        - contacting the [.NET Product Construction Services team via e-mail](mailto:dotnetprodconsvcs@microsoft.com).
+        """;
+
+
+    public override async Task<MergePolicyEvaluationResult> EvaluateAsync(PullRequestUpdateSummary pr, IRemote darc)
```

Comment:
it's probably something that was in the code in the old merge policies, but calling `IRemote` `darc` is very weird right?